code/modules/security_levels/security_levels.dm

```diff
@@ -81,6 +81,7 @@ GLOBAL_VAR_INIT(security_level, SEC_LEVEL_GREEN)
 					playsound(D, 'sound/machines/boltsup.ogg', 50, TRUE)
 		SSblackbox.record_feedback("tally", "security_level_changes", 1, get_security_level())
 		SSnightshift.check_nightshift()
+		sunset_get_atc_to_fuck_off(level) // sunset

```

Comment:
Name is confusing,you would expect get_atc_to_fuck_off to actually mute ATC yet this runs a check

Perhaps sunset_check_atc_security_level?